maven-surefire-plugin/src/site/apt/examples/junit-platform.apt.vm

```diff
@@ -411,6 +411,42 @@ Using JUnit 5 Platform
 </dependencies>
 +---+
 
+** JUnit5 Suite
+
+  For more information see this
+  {{{https://github.com/apache/maven-surefire/tree/master/surefire-its/src/test/resources/junit5-suite}example}}
+  and the {{{https://junit.org/junit5/docs/current/user-guide/#junit-platform-suite-engine}tutorial}}.
+
++---+
+<dependencies>
+    <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
+        <version>5.8.2</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.junit.platform</groupId>
+        <artifactId>junit-platform-suite-engine</artifactId>
+        <version>1.8.2</version>
+        <scope>test</scope>
+    </dependency>
+</dependencies>
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+                <includes>
+                    <include>JUnit5Tests</include>
```

Comment:
I afraid that users can simply comply such configurations .... and will be surprised

Reply:
Don't be surprised! What is the point of Junit5 Suite? It's the same point that the Junit4 suite has!
The Point is to start the suite class which is a similar case is in your Surefire project. The suite children are executed by selecting `JUnit5Tests` and not by surefire provider and that the reason you have to select it. You must select the suite and not the children. Why we do not use the parameter `test`? Because there is a significant difference between `includes` and `test`.  The `test` is a filter performing the selection inside of the JUnit which cannot be used here because this way you are not able to select the root class `JUnit5Tests` and you filter out all children. The `includes` does exactly what we want, it does not perform filtering inside of JUnit, it performs selection of classes that you `send` to the JUnit framework to execute it.

Reply:
such explanation should be put in example,
what inclusion is for and what value should contains.